### PR TITLE
chore: improve codebase

### PR DESCRIPTION
**Pull request by evilebottnawi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,133 @@
...
+.eclintercache
+.stylelintcache
+.sass-cache
+.phpcscache
+
+# npm
+npm-shrinkwrap.json
+!package-lock.json
+
+
```
Comment (TrySound) [inline .gitignore]: Does anything from this list appears in your project directory? This is trash IMO.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,3410 @@
+{
+  "name": "postcss-value-parser",
+  "version": "3.3.0",
```
Comment (TrySound) [inline package-lock.json]: I would prefer yarn.lock in this repo. npm has bad person as maintainer. I don't trust npm.

**Comment by evilebottnawi** [comment]
Feel free to feedback

**Comment by TrySound** [comment]
Wow, I'm not so fast to feedback in 30 seconds.

File: package-lock.json (hunk 2 above)
Comment (evilebottnawi) [reply package-lock.json]: @TrySound hm, let's do it :smile: Will send next PR

**Comment by evilebottnawi** [comment]
@TrySound I thought you lost all interest in the project so you decided to work by feedback, sorry

**Comment by TrySound** [comment]
Well, yes. I lost my interest in postcss. But I still care about web/node ecosystem.

File: .gitignore (hunk 1 above)
Comment (evilebottnawi) [reply .gitignore]: @TrySound it is standard .gitignore, `log`, `.idea`, `*.imp` and other trash

Comment (evilebottnawi) [reply .gitignore]: Also `.eslintcache` from `eslint`, i think better keep this list as is, there a lot of edge cases for difference os/editors/etc

**Pull request merged**